reliability-engineering/pipelines/concourse-deployer.yml

```diff
@@ -169,91 +168,72 @@ jobs:
     - get: tech-ops
       passed: [deploy]
       trigger: true
-    - get: infra
-      passed: [deploy]
     - get: concourse-release
       passed: [deploy]
       trigger: true
       params:
         globs:
         - concourse-*-linux-amd64.tgz.sha1
-  - task: configure-providers
-    file: tech-ops/reliability-engineering/pipelines/tasks/concourse-provider-config.yml
-    params:
-      DEPLOYMENT_NAME: ((deployment_name))
-  - task: get-current-workers
-    file: tech-ops/reliability-engineering/pipelines/tasks/concourse-get-workers.yml
-    params:
-      DEPLOYMENT_NAME: ((deployment_name))
-  - do:
-    - in_parallel:
-      - task: scale-out-web-node-asg
-        file: tech-ops/reliability-engineering/pipelines/tasks/asg-scale-max-capacity.yml
-        params:
-          ASG_PREFIX: ((deployment_name))-concourse-web
-      - task: scale-out-main-team-workers-asg
-        file: tech-ops/reliability-engineering/pipelines/tasks/asg-scale-max-capacity.yml
-        params:
-          ASG_PREFIX: ((deployment_name))-main-concourse-worker
-    - task: land-old-workers
-      file: tech-ops/reliability-engineering/pipelines/tasks/concourse-land-workers.yml
+  - in_parallel:
+    - task: refresh-web-node-asg
+      file: tech-ops/reliability-engineering/pipelines/tasks/asg-refresh-instances.yml
+      output_mapping:
+        refresh: web-refresh
       params:
-        DEPLOYMENT_NAME: ((deployment_name))
-    on_failure:
-      put: infra # try to scale back in on failure to stay in sync
-      attempts: 10
+        ASG_NAME: ((deployment_name))-concourse-web
+    - task: refresh-main-team-workers-asg
+      file: tech-ops/reliability-engineering/pipelines/tasks/asg-refresh-instances.yml
+      output_mapping:
+        refresh: main-team-worker-refresh
```

Comment:
we write to the `refresh` volume and try to read from it in a separate job.  To fix this, combine the `refresh-instances` and `wait-for-refresh` jobs into one job with different steps.